lisa/tests/kernel/hotplug/torture.py

```diff
@@ -27,6 +28,9 @@
 from lisa.env import TestEnv
 from lisa.utils import ArtifactPath
 
+class CPUHPSequenceError(Exception):
+    pass
+
 class HotplugTorture(TestBundle):
```

Comment:
If we're going to extend this to execute other hotplug tests, I would suggest to create a `HotplugTest` parent class, and have `HotplugTorture` as a child. Also, `_random_cpuhp_seq` should be renamed to a more generic name such as `_hotplug_sequence` or something along those lines. I guess that function should raise `NotImplementedError` in the `HotplugTest`, and be overridden by `HotplugTorture`, or something like this.